atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -124,88 +167,31 @@ def _render_mail(self, rebuild, success, canceled):
         ])
 
         endstate = None
-        if canceled:
-            endstate = 'canceled'
+        if auto_canceled or manual_canceled:
+            endstate = 'Canceled'
         else:
-            endstate = 'successful' if success else 'failed'
+            endstate = 'Succeeded' if success else 'Failed'
+
         url = None
-        if self.url and self.workflow.openshift_build_selflink:
-            url = urljoin(self.url, self.workflow.openshift_build_selflink + '/log')
+        try:
+            url = self._get_logs_url_from_build()
+        except Exception:
+            self.log.exception("Failed to fetch logs from koji")
+            if self.url and self.workflow.openshift_build_selflink:
+                url = urljoin(self.url, self.workflow.openshift_build_selflink + '/log')
+
+        image = self.workflow.image
+        if len(self.workflow.tag_conf.unique_images) > 0:
+            image = self.workflow.tag_conf.unique_images[0]
```

Comment:
This doesn't seem to work as expected in integration tests (it outputs the same image as `self.workflow.image`). Removed it for now - it would be introduced in the next PRs